### DISCUSSION TITLE
Pybricks Remote RGB Mode?

### DISCUSSION THREAD

**Discussion by Vinz1911** [pybricks/support#447] [Q&A] [answered]
Hello Again 👋 

Today I looked a bit inside the source code of the remote implementation and I saw that you enabled the RGB Mode on the Handset and change the color. This is very interesting. I tried to reproduce the byte sequence to change the led of the handset, but I got stucked, maybe you can explain me what I did wrong. 

I connected the remote with the nRF Connect app. 
I send the following command to enable RGB mode:
`[0x0A, 0x00, 0x41, 0x52, 0x01, 0x01, 0x00, 0x00, 0x00, 0x00]`

then I tried to change the color with the following command
`[0x0A, 0x00, 0x81, 0x52, 0x00, 0x00, 0x51, 0x01, 0x0A, 0xFF]`

i also looked into the LWP3.0 Documentation for the port output command. I'm also not sure with the length that's a bit confusing for me, maybe you can explain me what I did wrong here.

thank you

**Comment by dlech** [comment, marked as answer]
>  `[0x0A, 0x00, 0x81, 0x52, 0x00, 0x00, 0x51, 0x01, 0x0A, 0xFF]`

I think it should be:

```python
[
    0x0A, # length
    0x00, # hub
    0x81, # message type
    0x34, # port
    0x00, # startup and completion
    0x51, # command
    0x01, # mode
    0xXX, # red
    0xXX, # green
    0xXX, # blue
]
```

The port is 52 decimal, so 0x34 hex. And the startup and completion are combined in one byte.
